### DISCUSSION TITLE
Question About DatabaseBackup Proc

### DISCUSSION THREAD

**Discussion by Nehamnsu** [olahallengren/sql-server-maintenance-solution#444] [Questions & Answers] [answered]
DatabaseBackup Proc: We have Always On Enabled and are taking backups on Primary node. We are not explicitly setting the @CopyOnly to “Y” in our secondary node sql backup job. It’s just taking the default value in the stored procedure as “N”.  Will you advise us to include the @CopyOnly parameter to “Y” in the secondary node or does it automatically know that it’s the secondary node and it’s supposed to ignore taking backups on secondary? 

![image](https://user-images.githubusercontent.com/74800344/99862057-3a912900-2b4d-11eb-9dbd-3db984cf32f1.png)

**Comment by olahallengren** [comment, marked as answer]
@Nehamnsu, if you prefer to perform your backups on the primary replica, then you don't need to specify ```@CopyOnly = 'Y'```.

I have some information about backup of availability groups in the Frequently Asked Questions:
"How should I configure DatabaseBackup to back up an Availability Group?"
https://ola.hallengren.com/frequently-asked-questions.html

**Comment by Nehamnsu** [comment]
thank you
